roles/splunk_common/tasks/install_splunk.yml

```diff
@@ -10,6 +10,17 @@
   - "{{ manifests.files }}"
   when: splunk_upgrade | bool
 
+- name: Remove old python files
```

Comment:
LGTM - only comment would be that we might be able to merge this with the "Remove old manifest files" task above?

Reply:
I was talking with @lephino and we think we can use the old manifest to dictate what files to remove. This is likely a slightly larger change though so I would like to merge this in, so that we have a couple days and can resolve this issue from blocking others.